docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Do we have a company-wide code of conduct?

Reply:
No.